cpp-package/example/lenet.cpp

```diff
@@ -88,9 +83,8 @@ class Lenet {
     int val_fold = 1;
     int W = 28;
     int H = 28;
-    int batch_size = 42;
-    float learning_rate = 1e-4;
-    float weight_decay = 1e-4;
+    int batch_size = 64;
+    float learning_rate = 0.05;
```

Comment:
Is there any reference for you to choose these 2 parameters?

Reply:
The values are the defaults from the Python example.